plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/schema/confluent/ConfluentSchemaRegistryConfig.java

```diff
@@ -34,6 +35,8 @@
 public class ConfluentSchemaRegistryConfig
 {
     private Set<HostAddress> confluentSchemaRegistryUrls = ImmutableSet.of();
+    private String confluentSchemaRegistryUsername;
+    private String confluentSchemaRegistryPassword;
```

Comment:
Can we have a dedicated config class for it ? And also an enum to chose the authentication type - so that we could additional authentication mechanism in the future.